xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -489,6 +489,14 @@ $displayName##
  *#
 #macro(livetablecallback $divid $collist $colprops $transprefix)
 function (row, i, table) {
+   ## Make sure the "doc.location" property output is HTML in case the collist contains "doc.location"
+   #if($collist != $NULL && $collist.indexOf("doc.location") >=0)
+     #if (!$colprops["doc.location"])#set ($colprops["doc.location"] = {})#end
+     #if ($colprops["doc.location"].html == $NULL)
+       #set ($colprops["doc.location"].html = true)
```

Comment:
You're calling ``$colprops["doc.location"]`` 4 times. I would put it in a separate variable.